src/supabase/queries/auth.ts

```diff
@@ -1,3 +1,5 @@
+import { User } from '../../types/types';
+import supabaseAdmin from '../createAdminClient';
 import supabase from '../createClient';
 
 export const signUpUser = async (
```

Comment:
Let's update the `signUpUser` function to add address fields to the auth table rather than full address

Reply:
comments addressed: changed signupuser to full name and various address fields, router issue fixed, will focus on redundant querying after auth context implemented 💃